### PR TITLE
docs(Icon): updating documentation

### PR DESCRIPTION
**Pull request by LinKCoding**
### Overview
Reorganizes documentation for icon creation in storybook

Updating Icon documentation 

### PR Checklist

- [ ] Related to designs: 
- [x] Related to JIRA ticket: [GM-547](https://skillsoftdev.atlassian.net/browse/GM-547)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

[GM-547]: https://skillsoftdev.atlassian.net/browse/GM-547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.21.2-alpha.862fa5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://665f80b4461e2052a31db098--gamut-preview.netlify.app>https://665f80b4461e2052a31db098--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/665f80b4461e2052a31db098>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.29.5-alpha.21b673.0<br/></pre>

File: packages/styleguide/stories/Atoms/Icons/Mini.stories.mdx [hunk 1]
```diff
@@ -26,6 +26,14 @@ import {
   args={{}}
 />
 
+## General Infomation
+
+For mini icons, we use a subset of [Streamline's "Core Solid" icons](https://www.streamlinehq.com/icons/core-solid).
```
Comment (LinKCoding) [inline packages/styleguide/stories/Atoms/Icons/Mini.stories.mdx:31]: Check with Stacey if this link is ok

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx [hunk 2]
```diff
@@ -46,25 +46,12 @@ import {
 
 ## General Infomation
 
-We use a subset of [streamline icons](https://app.streamlineicons.com/streamline-regular), mainly the Outline version of the Regular set.
+For regular icons, we use a subset of [Streamline's "Ultimate Regular" icons](https://www.streamlinehq.com/icons/streamline-regular).
```
Comment (LinKCoding) [inline packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:49]: Check with Stacey if this link is ok (and removal of the clause is ok) seems like Regular don't use Outline? or shouldn't?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.29.5-alpha.752517.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://668d5dc33693ab94a9089f25--gamut-preview.netlify.app>https://668d5dc33693ab94a9089f25--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/668d5dc33693ab94a9089f25>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.29.5-alpha.c8f83a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://668d8a953efc2cd93975ab70--gamut-preview.netlify.app>https://668d8a953efc2cd93975ab70--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/668d8a953efc2cd93975ab70>Deploy Logs</a></p>

File: packages/styleguide/stories/Atoms/Icons/About.stories.mdx [hunk 3]
```diff
@@ -7,9 +7,39 @@ import { TableOfContents } from '~styleguide/blocks';
...
   }}
 />
 
+## Adding an Icon
+
+### Designers
+
+1. Identify a need for an icon.
+2. Finds/requests one on [streamline](https://www.streamlinehq.com/) (credentials in 1Password) for the appropriate type (Regular or Mini).
+3. Follows the Design guidelines for adding an Icon to the Figma library.
```
Comment (aresnik11) [inline packages/styleguide/stories/Atoms/Icons/About.stories.mdx:21]: should finds/requests/follows be singular here?

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx [hunk 4]
```diff
@@ -46,25 +46,12 @@ import {
...
 
-We like to standardize our icons by using a viewBox size of `"0 0 24 24"`. Icons should be centered and take up the entirety of this viewBox.
+Our icons by using standardized with a viewBox size of `"0 0 24 24"`. Icons should be centered and take up the entirety of this viewBox.
```
Comment (aresnik11) [inline packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:51]: Should this say "Our icons are standardized with..."?

File: packages/styleguide/stories/Atoms/Icons/About.stories.mdx (hunk 3 above)
Comment (LinKCoding) [reply packages/styleguide/stories/Atoms/Icons/About.stories.mdx:21]: Yea... not sure what I was thinking here, thanks!

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx (hunk 4 above)
Comment (LinKCoding) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:51]: yep! thanks!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.29.5-alpha.55e66a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://668ece4c2af68323b8a1d341--gamut-preview.netlify.app>https://668ece4c2af68323b8a1d341--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/668ece4c2af68323b8a1d341>Deploy Logs</a></p>

**Review by aresnik11 — APPROVED** [review]
LGTM!

**Pull request merged**